pom.xml

```diff
@@ -705,7 +706,7 @@
       <dependency>
         <groupId>org.apache.curator</groupId>
         <artifactId>curator-recipes</artifactId>
-        <version>2.4.0</version>
+        <version>${curator.version}</version>
```

Comment:
LGTM but do we need to explicitly manage `curator-client` versions too for consistency? just manage, not actually make a dependency on anywhere.